### PR TITLE
dev-util/android-studio: bump to 3.3.1.0.182.5264788

### PR DESCRIPTION
**Pull request by pavelhomolka**
Bug: https://bugs.gentoo.org/677462
Signed-off-by: Pavel Homolka <pavelhomolka1986@gmail.com>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

**Comment by oniGino** [comment]
Latest build is now  3.4.0.18.183.5452501 FYI

File: dev-util/android-studio/android-studio-3.3.1.0.182.5264788.ebuild [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	>=x11-libs/libXrender-0.9.8
+	>=x11-libs/libXxf86vm-1.1.3
+	>=x11-libs/libdrm-2.4.46
+	>=x11-libs/libxcb-1.9.1
+	>=x11-libs/libxshmfence-1.1"
+S=${WORKDIR}/${PN}
+PATCHES=( "${FILESDIR}/0001-use-java-home-before-bundled.patch" )
+
+src_prepare() {
+	eapply "${PATCHES[@]}"
```
Comment (juippis) [inline dev-util/android-studio/android-studio-3.3.1.0.182.5264788.ebuild]: You can just call `default` in here. It applies everything in `PATCHES` array, and also calls `eapply_user`. So you can then remove the eapply_user part below.

File: dev-util/android-studio/android-studio-3.3.1.0.182.5264788.ebuild [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	doins -r *
+
+	# Replaced bundled jre with system vm/jdk
+	# This is really a bundled jdk not a jre
+	rm -rf "${ED%/}${dir}/jre" || die
+	dosym "../../etc/java-config-2/current-system-vm" "${dir}/jre"
+
+	fperms 755 "${dir}/bin/studio.sh" "${dir}"/bin/fsnotifier{,64}
+	fperms -R 755 "${dir}"/bin/lldb/{android,bin}
+	chmod 755 "${ED%/}${dir}"/bin/*.py "${ED%/}${dir}"/bin/*.sh || die
```
Comment (juippis) [inline dev-util/android-studio/android-studio-3.3.1.0.182.5264788.ebuild]: Why not use `fperms` here as well, like you do above? You then don't have to append `|| die` either. Same applies to the line below.

(@gyakovlev)

**Comment by perfinion** [comment]
I bumped to 3.4.0 just now, thanks for this tho!

**Pull request closed without merging**